cpp/src/community/leiden_impl.cuh

```diff
@@ -14,19 +14,23 @@
  * limitations under the License.
  */
 #pragma once
+// #define TIMING
+
+#include <community/detail/common_methods.hpp>
+#include <community/detail/refine.hpp>
 #include <community/flatten_dendrogram.hpp>
-#include <cugraph/algorithms.hpp>
-#include <cugraph/graph_view.hpp>
-#include <cugraph/utilities/error.hpp>
-#include <cugraph/vertex_partition_device_view.cuh>
+#include <cugraph/detail/shuffle_wrappers.hpp>
+#include <cugraph/detail/utility_wrappers.hpp>
+#include <cugraph/graph.hpp>
+#include <cugraph/utilities/high_res_timer.hpp>
+#include <prims/update_edge_src_dst_property.cuh>
```

Comment:
Headers under `cpp\include` and internal headers (`cpp\src`) need to be separated by space.